src/pages/products/testing-center/index.tsx

```diff
@@ -0,0 +1,181 @@
+            </ValuePropCard>
+            <ValuePropCard title="Student-centric exam scheduling">
+              Empower students, especially those balancing family and work
+              obligations, with the flexibility to schedule exams at their
+              preferred times within the exam period. Enable easy reservation
+              modifications and cancellations up to the selected exam time,
+              providing convenience and autonomy.
+            </ValuePropCard>
+            <ValuePropCard title="Seamless exam overrides">
+              Empower instructors with the capability to effortlessly grant
+              overrides for students facing unexpected circumstances, such as
+              illness or unavoidable absences during the exam period.
+            </ValuePropCard>
+            <ValuePropCard title="Authentic testing">
+              Create an authentic testing environment by providing students with
+              access to computers equipped with compilers, debuggers, and
+              graphical software, ensuring a comprehensive assessment of their
+              skills and knowledge.
+            </ValuePropCard>
+          </div>
+        </div>
+      </div>
+
+      <div className={classnames("container-fluid py-4", styles.container)}>
+        <div className="container-md">
+          <div className="row">
+            <div className="col-md-6 order-1 pt-4">
+              <Image
+                src={reducedDistractionImage}
+                alt="seat in reduced distraction environment"
+                style={{
+                  maxWidth: "100%",
+                  height: "auto",
+                }}
+              />
+            </div>
+            <div className="col-md-6 order-2 pt-4">
+              <div className="row">
+                <Heading>Academic accommodations</Heading>
+              </div>
+              <p>
+                Students with documented academic accommodations can take the
+                exams in the same setting as their classmates, by using special
+                seats inside the testing center. Below are some common requests
+                for accommodations that can be arranged:
+              </p>
+              <ul>
+                <li>Extended exam time</li>
+                <li>Reduced distraction environment</li>
+                <li>Text-to-speech software</li>
+                <li>
+                  Accessible seating for students with wheelchairs or other
+                  mobility needs
+                </li>
+                <li>Special arrangements for bathroom breaks</li>
+                <li>Access to disability related technology or equipment</li>
+                <li>Special/adjustable furniture</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <BannerCTA
+        title="Need more information?"
+        subtitle="Are you interested in building a testing center? We can help!"
```

Comment:
Nit, since the title already includes a question.

```suggestion
        subtitle="If you're interested in building a testing center, we can help!"
```